built_value_generator/lib/src/value_source_class.dart

```diff
@@ -209,6 +249,11 @@ abstract class ValueSourceClass
       }
     }
 
+    if (extendsNonObject) {
+      result.add('Stop class extending other classes. '
```

Comment:
Looks like there is a "using" missing after "Stop"

Reply:
"Class" is supposed to refer to the value class, as opposed to the builder. But I agree it's unclear. I filed #320 to improve this, I think it would be better to always include the class/builder name.

I'm probably going to drop this PR anyway, need to split it into smaller pieces and do some related fixes first.

Thanks :)

Reply:
Ok, got it now. Was quite early in the morning when I looked though